data/data.go

```diff
@@ -5,17 +5,28 @@ import (
 	"fmt"
 )
 
+// ListKey represents the primary key of a list
+type ListKey struct {
+	ID string `json:"Id"`
```

Comment:
If we are making the variable same capitalisation on both `i` and `d`, should we do the same in the json? either:
`id` or `ID`

Reply:
This keeps it consistent with the other json representations we have, and changing this now would be a breaking change for the client (not that we have many users to break), and a breaking change for the DB (we'd have to update the schema). I agree that the casing that we've gone for is odd though.